aws-resources/src/main/java/io/opentelemetry/contrib/aws/resource/CloudResourceProvider.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.aws.resource;
+
+import io.opentelemetry.sdk.autoconfigure.spi.ConfigProperties;
+import io.opentelemetry.sdk.autoconfigure.spi.internal.ConditionalResourceProvider;
+import io.opentelemetry.sdk.resources.Resource;
+import io.opentelemetry.semconv.ResourceAttributes;
+
+public abstract class CloudResourceProvider implements ConditionalResourceProvider {
```

Comment:
does this need to be public?

```suggestion
abstract class CloudResourceProvider implements ConditionalResourceProvider {
```

Reply:
@zeitlinger not sure if you missed this as it got marked as resolved?

Reply:
sorry - missed that